content/howto8/integration/implement-cicd-pipeline.md

```diff
@@ -39,7 +39,7 @@ To build the deployment package, use the [Team Server API](/apidocs-mxsdk/apidoc
 
 #### 3.2.1 Getting the Latest Revision
 
-To get the latest revision, use the [Retrieve Branch API call](/apidocs-mxsdk/apidocs/team-server-api#retrieve-branch) to get the `LatestRevisionNumber` from the output.
+To get the latest revision, use the [Retrieve Branch API call](/apidocs-mxsdk/apidocs/old-version/team-server-api#retrieve-branch) to get the `LatestRevisionNumber` from the output.
```

Comment:
Same comment, let's use the new documentation.

Reply:
The link now points to the new APIs

Reply:
Removed